pkg/oci/containerd.go

```diff
@@ -475,3 +504,33 @@ func getHostFile(fs afero.Fs, configPath string, appendToBackup bool, registryUR
 	}
 	return hf, false, nil
 }
+
+// getOrderedHosts parses the given byte slice as TOML data and returns a slice of
+// strings containing the ordered hosts found in the TOML data.
+func getOrderedHosts(b []byte) []string {
```

Comment:
Do we still need this? Could this be asserted through string comparison in tests?

Reply:
Sorry, I don't really understand what you mean.
You need to get the hosts in order to be able to rebuild the config file with the same order of hosts as previously (with spegel hosts added).